libs/features/merch.js

```diff
@@ -0,0 +1,66 @@
+import { loadScript, getConfig, createTag } from '../utils/utils.js';
+
+const { miloLibs, codeRoot, env } = getConfig();
+const base = miloLibs || codeRoot;
+
+export function decorateButton(osi, a, text) {
+  if (!osi || !a) return;
+  a.href = '#';
+  a.dataset.checkoutClientid = 'mini_plans';
```

Comment:
should this be configurable?  Or hard-coded just for POC?

Reply:
at the moment still hard-coded, since there's so much that's still foggy for us. 👍